packages/bpk-component-barchart/README.md

```diff
@@ -49,7 +49,7 @@ export default () => (
 | className                               | string                                | false    | null             |
 | leadingScrollIndicatorClassName         | string                                | false    | null             |
 | trailingScrollIndicatorClassName        | string                                | false    | null             |
-| [outlierPercentage](#outlierpercentage) | number                                | false    | null             |
+| [`outlierPercentage`](#outlierpercentage) | number                                | false    | null             |
```

Comment:
Why should these have code style when those that aren't likes don't

Reply:
Ideally, *all* references to code should be in backticks and then we wouldn't need that huge list of terms in `.spelling` because it ignores anything in backticks. In this specific case I was using the interactive CLI to escape other uses of this and it offers to change others to the same format, which I must have accidentally pressed. I'll change them back.